superset/typing.py

```diff
@@ -42,7 +42,7 @@ class AdhocMetric(TypedDict):
 ]
 DbapiDescription = Union[List[DbapiDescriptionRow], Tuple[DbapiDescriptionRow, ...]]
 DbapiResult = Sequence[Union[List[Any], Tuple[Any, ...]]]
-FilterValue = Union[datetime, float, int, str]
+FilterValue = Union[bool, datetime, float, int, str]
```

Comment:
boolean type was missing here

Reply:
is there a test for this filter value?

Reply:
Not really - these are just handled by SqlAlchemy, so not that much to test (they just work if the column type is correct).